.github/workflows/build.yml

```diff
@@ -42,6 +44,18 @@ jobs:
         run: sudo apt-get update && sudo apt-get install gcc-aarch64-linux-gnu -y
         if: matrix.target == 'aarch64-unknown-linux-gnu'
 
+      - name: Install protoc
```

Comment:
If we're checking in the generated files then do we need protoc?

Reply:
No, I don't think so. I removed it from GH actions and moved the proto generation out of the build script into a separate binary